### PR TITLE
Femtoworld

### PR DESCRIPTION
**Pull request by zchochul**
* Changed printfs and couts for LOGFs and LOGs
* Fixing filling phi candidates table filling

### CODE REVIEW

File: PWGCF/FemtoWorld/Core/FemtoWorldDetaDphiStar.h [hunk 1]
```diff
@@ -68,6 +68,18 @@ class FemtoWorldDetaDphiStar
...
+      /*for (int i = 0; i < 2; i++) {
+        std::string dirName = static_cast<std::string>(dirNames[1]);
+        histdetadpi[i][0] = mHistogramRegistry->add<TH2>((dirName + static_cast<std::string>(histNames[0][i])).c_str(), "; #Delta #eta; #Delta #phi", kTH2F, {{100, -0.15, 0.15}, {100, -0.15, 0.15}});
+        histdetadpi[i][1] = mHistogramRegistry->add<TH2>((dirName + static_cast<std::string>(histNames[1][i])).c_str(), "; #Delta #eta; #Delta #phi", kTH2F, {{100, -0.15, 0.15}, {100, -0.15, 0.15}});
+        if (plotForEveryRadii) {
+          for (int j = 0; j < 9; j++) {
+            histdetadpiRadii[i][j] = mHistogramRegistryQA->add<TH2>((dirName + static_cast<std::string>(histNamesRadii[i][j])).c_str(), "; #Delta #eta; #Delta #phi", kTH2F, {{100, -0.15, 0.15}, {100, -0.15, 0.15}});
+          }
+        }
+      }*/
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Core/FemtoWorldDetaDphiStar.h:81]: Please remove comments
You can test your code locally until you are happy with it and then push it but don't leave pending code commented

File: PWGCF/FemtoWorld/Core/FemtoWorldDetaDphiStar.h [hunk 2]
```diff
@@ -114,6 +126,30 @@ class FemtoWorldDetaDphiStar
...
+         auto deta = part1.eta() - daughter.eta();
+         auto dphiAvg = AveragePhiStar(part1, *daughter, i);
+         histdetadpi[i][0]->Fill(deta, dphiAvg);
+         if (pow(dphiAvg, 2) / pow(deltaPhiMax, 2) + pow(deta, 2) / pow(deltaEtaMax, 2) < 1.) {
+           pass = true;
+         } else {
+           histdetadpi[i][1]->Fill(deta, dphiAvg);
+         }
+       }
+       */
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Core/FemtoWorldDetaDphiStar.h:151]: Same comment as above

File: PWGCF/FemtoWorld/Core/FemtoWorldPhiSelection.h [hunk 3]
```diff
@@ -387,8 +385,8 @@ bool FemtoWorldPhiSelection::isSelectedMinimal(C const& col, V const& v0, T cons
   const auto signPos = posTrack.sign();
   const auto signNeg = negTrack.sign();
   if (signPos < 0 || signNeg > 0) {
-    printf("-Something wrong in isSelectedMinimal--\n");
-    printf("ERROR - Wrong sign for Phi daughters\n");
+    LOGF(info, "-Something wrong in isSelectedMinimal--\n");
+    LOGF(info, "ERROR - Wrong sign for Phi daughters\n");
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Core/FemtoWorldPhiSelection.h:389]: Consider using
```
LOGF(error, "....);
```

File: PWGCF/FemtoWorld/Core/FemtoWorldPhiSelection.h [hunk 4]
```diff
@@ -459,8 +457,8 @@ void FemtoWorldPhiSelection::fillPhiQA(C const& col, V const& v0, T const& posTr
...
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Core/FemtoWorldPhiSelection.h:461]: Same as above

File: PWGCF/FemtoWorld/Core/FemtoWorldPhiSelection.h [hunk 5]
```diff
@@ -613,9 +611,9 @@ void FemtoWorldPhiSelection::fillQA(C const& col, V const& v0, T const& posTrack
     TLorentzVector sumVec(part1Vec);
     sumVec += part2Vec;
 
-    //float phiPx = posTrack.px() + negTrack.px();
-    //float phiPy = posTrack.py() + negTrack.py();
-    //float phiPz = posTrack.pz() + negTrack.pz();
+    // float phiPx = posTrack.px() + negTrack.px();
+    // float phiPy = posTrack.py() + negTrack.py();
+    // float phiPz = posTrack.pz() + negTrack.pz();
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Core/FemtoWorldPhiSelection.h:616]: Please, don't leave code commented

File: PWGCF/FemtoWorld/Core/FemtoWorldV0Selection.h [hunk 6]
```diff
@@ -323,8 +321,8 @@ bool FemtoWorldV0Selection::isSelectedMinimal(C const& col, V const& v0, T const
   const auto signPos = posTrack.sign();
   const auto signNeg = negTrack.sign();
   if (signPos < 0 || signNeg > 0) {
-    printf("-Something wrong in isSelectedMinimal--\n");
-    printf("ERROR - Wrong sign for V0 daughters\n");
+    LOGF(info, "-Something wrong in isSelectedMinimal--\n");
+    LOGF(info, "ERROR - Wrong sign for V0 daughters\n");
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Core/FemtoWorldV0Selection.h:325]: Same comment as above

File: PWGCF/FemtoWorld/Tasks/CMakeLists.txt [hunk 7]
```diff
@@ -33,8 +33,9 @@ o2physics_add_dpl_workflow(femto-world-pair-track-phi
...
           COMPONENT_NAME Analysis)
-o2physics_add_dpl_workflow(femto-world-pair-track-v0
-          SOURCES femtoWorldPairTaskTrackV0.cxx
-          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
-          COMPONENT_NAME Analysis)
+          
+#o2physics_add_dpl_workflow(femto-world-pair-track-v0
+#          SOURCES femtoWorldPairTaskTrackV0.cxx
+#          PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore
+#          COMPONENT_NAME Analysis)
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Tasks/CMakeLists.txt:40]: No comments left

File: PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackTrack.cxx [hunk 8]
```diff
@@ -281,7 +279,6 @@ struct femtoWorldPairTaskTrackTrack {
                     phiEta,
                     phiPhi);
         tmpIDtrack.push_back(p1.globalIndex());
-        std::cout<< "phi przeszło" << std::endl;
         //Partition<aod::FemtoWorldPhiCandidates> Phiparts = (aod::femtoworldparticle::eta < cfgEtaHighPart2) && (aod::femtoworldparticle::eta > cfgEtaLowPart2);
         //sameEventCont.setPair(p1, phi multCol);
       }
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackTrack.cxx:284]: Please remove commented code

**Review by victor-gonzalez — CHANGES_REQUESTED** [review]
Please, fix the formatting errors and implement the suggested changes

File: PWGCF/FemtoWorld/Core/FemtoWorldContainer.h [hunk 9]
```diff
@@ -151,6 +152,7 @@ class FemtoWorldContainer
     }
 
     if (mHistogramRegistry) {
+      // LOGF(info, "O KURDE WITAM =========================FILLING======================================== \n"); WORKING
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Core/FemtoWorldContainer.h:155]: Same comment as above

File: PWGCF/FemtoWorld/DataModel/FemtoWorldDerived.h [hunk 10]
```diff
@@ -98,11 +99,11 @@ DECLARE_SOA_DYNAMIC_COLUMN(Pz, pz, //! Compute the momentum in z in GeV/c
...
-DECLARE_SOA_DYNAMIC_COLUMN(P, p, //! Compute the overall momentum in GeV/c
-                           [](float pt, float eta) -> float {
-                             return pt * std::cosh(eta);
-                           });
-// debug variables
+// DECLARE_SOA_DYNAMIC_COLUMN(P, p, //! Compute the overall momentum in GeV/c
+//                            [](float pt, float eta) -> float {
+//                              return pt * std::cosh(eta);
+//                            });
+//  debug variables
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/DataModel/FemtoWorldDerived.h:106]: Please, no commented code

File: PWGCF/FemtoWorld/DataModel/FemtoWorldDerived.h [hunk 11]
```diff
@@ -144,7 +147,7 @@ DECLARE_SOA_TABLE(FemtoWorldParticles, "AOD", "FEMTOWORLDPARTS",
                   femtoworldparticle::Px<femtoworldparticle::Pt, femtoworldparticle::Phi>,
                   femtoworldparticle::Py<femtoworldparticle::Pt, femtoworldparticle::Phi>,
                   femtoworldparticle::Pz<femtoworldparticle::Pt, femtoworldparticle::Eta>,
-                  femtoworldparticle::P<femtoworldparticle::Pt, femtoworldparticle::Eta>, // tu koniec poprzedniej wersji
+                  // femtoworldparticle::P<femtoworldparticle::Pt, femtoworldparticle::Eta>, // tu koniec poprzedniej wersji
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/DataModel/FemtoWorldDerived.h:150]: Please, no commented code

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 12]
```diff
@@ -56,6 +59,12 @@ using FemtoFullTracks = soa::Join<aod::FullTracks,
                                   aod::pidTPCKa, aod::pidTPCPr, aod::pidTPCDe,
                                   aod::pidTOFEl, aod::pidTOFMu, aod::pidTOFPi,
                                   aod::pidTOFKa, aod::pidTOFPr, aod::pidTOFDe, aod::pidTOFbeta>;
+using FemtoPhiTracks = soa::Join<aod::FullTracks,
+                                 aod::TracksDCA, aod::TOFSignal,
+                                 aod::pidTPCEl, aod::pidTPCMu, aod::pidTPCPi,
+                                 aod::pidTPCKa, aod::pidTPCPr, aod::pidTPCDe,
+                                 aod::pidTOFEl, aod::pidTOFMu, aod::pidTOFPi,
+                                 aod::pidTOFKa, aod::pidTOFPr, aod::pidTOFDe, aod::pidTOFbeta>;
 // using FilteredFullV0s = soa::Filtered<aod::V0Datas>; /// predefined Join table for o2::aod::V0s = soa::Join<o2::aod::TransientV0s, o2::aod::StoredV0s> to be used when we add v0Filter
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:68]: Please, no commented code

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 13]
```diff
@@ -384,16 +391,18 @@ struct femtoWorldProducerTask {
       }
       trackCuts.fillQA<aod::femtoworldparticle::ParticleType::kTrack, aod::femtoworldparticle::TrackType::kNoChild>(track);
       // the bit-wise container of the systematic variations is obtained
-      auto cutContainer = trackCuts.getCutContainer<aod::femtoworldparticle::cutContainerType>(track);
+      // auto cutContainer = trackCuts.getCutContainer<aod::femtoworldparticle::cutContainerType>(track);
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:394]: Please, no commented code

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 14]
```diff
@@ -484,106 +493,107 @@ struct femtoWorldProducerTask {
...
+                       postrack.tofNSigmaStoreKa(),
+                       postrack.tofNSigmaStorePr(),
+                       postrack.tofNSigmaStoreDe(),
+                       -999.,
+                       -999.,
+                       -999.,
+                       -999.,
+                       -999.,
+                       -999.);*/
+          // const int rowOfPosTrack = outputParts.lastIndex();
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:545]: Same as above

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 15]
```diff
@@ -484,106 +493,107 @@ struct femtoWorldProducerTask {
...
+                       -999.,
+                       -999.,
+                       -999.,
+                       -999.);*/
+          // const int rowOfNegTrack = outputParts.lastIndex();
+          // int indexChildID[2] = {rowOfPosTrack, rowOfNegTrack};
+          /*outputParts(outputCollision.lastIndex(),
                       v0.pt(),
                       v0.eta(),
                       v0.phi(),
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:599]: Same as above

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 16]
```diff
@@ -625,7 +635,7 @@ struct femtoWorldProducerTask {
                       -999.,
                       -999.,
                       -999.,
-                      -999.);
+                      -999.);*/
           /* if (ConfDebugOutput) {
              outputDebugParts(postrack.sign(),
                               (uint8_t)postrack.tpcNClsFound(),
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:641]: Same as above

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 17]
```diff
@@ -712,10 +722,259 @@ struct femtoWorldProducerTask {
...
+      Configurable<float> cfgEtaHighPart2{"cfgEtaHighPart2", 0.8, "Higher limit for Eta for the second particle"};
+      Configurable<float> cfgDcaXYPart2{"cfgDcaXYPart2", 2.4, "Value for DCA_XY for the second particle"};
+      Configurable<float> cfgDcaZPart2{"cfgDcaZPart2", 3.2, "Value for DCA_Z for the second particle"};
+      Configurable<int> cfgTpcClPart2{"cfgTpcClPart2", 88, "Number of tpc clasters for the second particle"};             // min number of found TPC clusters
+      Configurable<int> cfgTpcCrosRoPart2{"cfgTpcCrosRoPart2", 70, "Number of tpc crossed rows for the second particle"}; // min number of crossed rows
+      Configurable<float> cfgChi2TpcPart2{"cfgChi2TpcPart2", 4.0, "Chi2 / cluster for the TPC track segment for the second particle"};
+      Configurable<float> cfgChi2ItsPart2{"cfgChi2ItsPart2", 36.0, "Chi2 / cluster for the ITS track segment for the second particle"};
+
+      for (auto& [p1, p2] : combinations(soa::CombinationsStrictlyUpperIndexPolicy(tracks, tracks))) {
+        // LOGF(info, "p1.type = %u \n p2.type = %u \n", p1.trackType(), p2.trackType());
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:757]: Same as above

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 18]
```diff
@@ -712,10 +722,259 @@ struct femtoWorldProducerTask {
...
+          }
+
+        } else if ((p2.p() <= 0.45f)) {
+          if (!(IsKaonTPCdEdxNSigma(p2.p(), p2.tpcNSigmaKa()))) {
+            continue;
+          }
+        }
+
+        // LOGF(info, "p1.globalIndex() = %u \t outputCollision.lastIndex() = %u \n", p1.globalIndex(), outputCollision.lastIndex());
+        // LOGF(info, "p2.globalIndex() = %u \t outputCollision.lastIndex() = %u \n", p2.globalIndex(), outputCollision.lastIndex());
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:795]: Same as above

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 19]
```diff
@@ -712,10 +722,259 @@ struct femtoWorldProducerTask {
...
+        TLorentzVector sumVec(part1Vec);
+        sumVec += part2Vec;
+
+        float phiEta = sumVec.Eta();
+        float phiPhi = sumVec.Phi();
+        float phiPt = sumVec.Pt();
+        float phiP = sumVec.P();
+        float phiM = sumVec.M();
+
+        // LOGF(info, "FIRST DAUGHTER: \n pT = %f \n eta = %f \n phi = %f", p1.pt(), p1.eta(), p1.phi());
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:813]: Same as above

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 20]
```diff
@@ -712,10 +722,259 @@ struct femtoWorldProducerTask {
...
+        PhiCuts.fillQA<aod::femtoworldparticle::ParticleType::kPhi, aod::femtoworldparticle::ParticleType::kPhiChild>(col, p1, p1, p2); ///\todo fill QA also for daughters
+        auto cutContainerV0 = PhiCuts.getCutContainer<aod::femtoworldparticle::cutContainerType>(col, p1, p2);
+        if (true) { // temporary true value, we are doing simpler version first
+          int postrackID = p1.globalIndex();
+          int rowInPrimaryTrackTablePos = -1;
+          rowInPrimaryTrackTablePos = getRowDaughters(postrackID, tmpIDtrack);
+          childIDs[0] = rowInPrimaryTrackTablePos;
+          childIDs[1] = 0;
+          // Printing info of first daughter (Kaon)
+          // LOGF(info, "FIRST DAUGHTER: \n pT = %f \n eta = %f \n phi = %f", p1.pt(), p1.eta(), p1.phi());
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:824]: Same as above

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 21]
```diff
@@ -712,10 +722,259 @@ struct femtoWorldProducerTask {
...
+                      -999.,
+                      -999.,
+                      -999.);
+          const int rowOfPosTrack = outputParts.lastIndex();
+          int negtrackID = p2.globalIndex();
+          int rowInPrimaryTrackTableNeg = -1;
+          rowInPrimaryTrackTableNeg = getRowDaughters(negtrackID, tmpIDtrack);
+          childIDs[0] = 0;
+          childIDs[1] = rowInPrimaryTrackTableNeg;
+          // LOGF(info, "SECOND DAUGHTER: \n pT = %f \n eta = %f \n phi = %f", p2.pt(), p2.eta(), p2.phi());
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:876]: Same as above

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 22]
```diff
@@ -712,10 +722,259 @@ struct femtoWorldProducerTask {
...
+      }
+    }
   }
   PROCESS_SWITCH(femtoWorldProducerTask, processProd, "Produce Femto tables", true);
   void processPhi(aod::FemtoFullCollision const& col, aod::BCsWithTimestamps const&, aod::FemtoFullTracks const& tracks) /// \todo with FilteredFullV0s
   {
+    /*
     // First particle
     Configurable<int> ConfPDGCodePartOne{"ConfPDGCodePartOne", 321, "Particle 1 - PDG code"};
     Configurable<float> cfgPtLowPart1{"cfgPtLowPart1", 0.14, "Lower limit for Pt for the first particle"};
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:980]: Same as above

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 23]
```diff
@@ -747,8 +1006,14 @@ struct femtoWorldProducerTask {
...
 
-      for (auto& [p1, p2] : combinations(tracks, tracks)) {
+      for (auto& [p1, p2] : combinations(soa::CombinationsFullIndexPolicy(tracks, tracks))) {
+        LOGF(info, "p1.pt() = %f \n p1.p() = %f \n", p1.pt(), p1.p());
+        LOGF(info, "p2.pt() = %f \n p2.p() = %f \n", p2.pt(), p2.p());
+        if (p1.globalIndex() == p2.globalIndex()) {
+          continue;
+        }
         if ((p1.pt() < cfgPtLowPart1) || (p1.pt() > cfgPtHighPart1)) {
+          //LOGF(info, "EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE", p1.pt());
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:1016]: Same as above

File: PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx [hunk 24]
```diff
@@ -946,11 +1220,10 @@ struct femtoWorldProducerTask {
                       -999.,
                       -999.);
         }
-        // std::cout<< "phi przeszło" << std::endl;
       }
     }
-  }
-  PROCESS_SWITCH(femtoWorldProducerTask, processPhi, "Produce Phi candidates tables", true);
+  */}
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/TableProducer/femtoWorldProducerTask.cxx:1225]: Same as above

File: PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackPhi.cxx [hunk 25]
```diff
@@ -128,29 +128,33 @@ struct femtoWorldPairTaskTrackPhi {
...
+    for (auto& part : groupPartsOne) { // jakby tu nie wchodzilo
+      /*
       if (part.p() > cfgCutTable->get("PartOne", "MaxP") || part.pt() > cfgCutTable->get("PartOne", "MaxPt")) {
         continue;
       }
       if (!isFullPIDSelected(part.pidcut(), part.p(), cfgCutTable->get("PartOne", "PIDthr"), vPIDPartOne, cfgNspecies, kNsigma, cfgCutTable->get("PartOne", "nSigmaTPC"), cfgCutTable->get("PartOne", "nSigmaTPCTOF"))) {
         continue;
-      }
+      }*/
+      // LOGF(info, "TEST ==============================TRACKONEFILL=================================== \n");
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackPhi.cxx:144]: Same as above

File: PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackPhi.cxx [hunk 26]
```diff
@@ -128,29 +128,33 @@ struct femtoWorldPairTaskTrackPhi {
...
       trackHistoPartOne.fillQA(part);
     }
     for (auto& part : groupPartsTwo) {
+      // LOGF(info, "TEST ==============================TRACKPHIFILL=================================== \n");
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackPhi.cxx:148]: Same as above

File: PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackPhi.cxx [hunk 27]
```diff
@@ -128,29 +128,33 @@ struct femtoWorldPairTaskTrackPhi {
...
     /// Now build the combinations
     for (auto& [p1, p2] : combinations(groupPartsOne, groupPartsTwo)) {
+      /*
       if (p1.p() > cfgCutTable->get("PartOne", "MaxP") || p1.pt() > cfgCutTable->get("PartOne", "MaxPt")) {
         continue;
       }
+
       if (!isFullPIDSelected(p1.pidcut(), p1.p(), cfgCutTable->get("PartOne", "PIDthr"), vPIDPartOne, cfgNspecies, kNsigma, cfgCutTable->get("PartOne", "nSigmaTPC"), cfgCutTable->get("PartOne", "nSigmaTPCTOF"))) {
         continue;
       }
```
Comment (victor-gonzalez) [inline PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackPhi.cxx:160]: Same as above

**Review by victor-gonzalez — CHANGES_REQUESTED** [review]
Please, get rid of all commented code
If you are requiring to merge you code it will go into production
If you are just keeping your changes in a central repository change your PR to draft

File: PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackPhi.cxx (hunk 27 above)
Comment (zchochul) [reply PWGCF/FemtoWorld/Tasks/femtoWorldPairTaskTrackPhi.cxx:158]: This was commented only temporarily. In next PR I will be adding cuts to the histograms

**Review by victor-gonzalez — APPROVED** [review]
I still see files with code commented
This is not a good practice for going into production
Commenting code for making it readable is OK. Leaving code commented is not, it makes difficult its interpretation
For the time being I approve the PR to move it forward. But please, spend some time in removing all commented lines of code

**Comment by zchochul** [comment]
I'll try to avoid leaving commented code. Thank you for understanding!

**Pull request merged**